### ISSUE TITLE
Error 'OpenVR version is outdated. Please update OpenVR.

### ISSUE THREAD

**Issue by alexvmirov** [matzman666/OpenVR-AdvancedSettings#31]
![image](https://cloud.githubusercontent.com/assets/24753662/21947972/0cbcae9a-d9e0-11e6-93f8-c6c37ca968e0.png)

Hi, I have been searching how to update OpenVR and have not found much information. Can anyone help?

Thanks.

**Comment by StewieGriffin** [comment]
Click "Code" tab , then "releases"

**Comment by matzman666** [comment]
OpenVR (better said SteamVR) can only be updated via Steam. What version of SteamVR are you running.

**Comment by Strgnoff** [comment]
Hi! I'm getting the same error.

Installed SteamVR from Steam library.
SteamVR is up to date according to Steam
Steam is installed in Default Folder (C:\Program Files (x86)\Steam)
I tried changing Steam language from German to English

Everything else in SteamVR is working perfectly fine.

Is it possible to Output a more verbose error message? (I can't compile by myself right now)

**Comment by hyperseven** [comment]
have you tried the beta branch of SteamVR?

**Comment by Strgnoff** [comment]
I will try beta branch now!

**Comment by Strgnoff** [comment]
Looks like I have permission/ownership issues inside my SteamApps Folder because I copy/pasted it from another Computer. Trying to fix it now and will Report back.

**Comment by Balderick** [comment]
Running steam as administrator is a recommendation made on steam support pages for steamvr.
https://support.steampowered.com/kb_article.php?ref=8566-SDZC-9326

**Comment by Strgnoff** [comment]
Good point, but wouldn't have helped in my case. In Windows even the administrator can't delete or edit files if they are owned by the administrator of another machine. Not before claiming ownership of said files. Does not make any sense? Yep, that's Windows! ;)
